cmd/vclusterctl/cmd/get/service_cidr.go

```diff
@@ -0,0 +1,121 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/flags"
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/log"
+	"github.com/loft-sh/vcluster/pkg/helm/values"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	errorMessageIPFamily     = "expected an IPv6 value as indicated by " // Dual-stack cluster with .spec.ipFamilies=["IPv6"]
+	errorMessageIPv4Disabled = "IPv4 is not configured on this cluster"  // IPv6 only cluster
+)
+
+type serviceCIDRCmd struct {
+	*flags.GlobalFlags
+	log log.Logger
+}
+
+func getServiceCIDR(globalFlags *flags.GlobalFlags) *cobra.Command {
+	cmd := &serviceCIDRCmd{
+		GlobalFlags: globalFlags,
+		log:         log.GetInstance(),
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:   "service-cidr",
+		Short: "Prints Service CIDR of the cluster",
+		Long: `
+#######################################################
+############### vcluster get service-cidr  ############
+#######################################################
+Prints Service CIDR of the cluster
+
+Ex: 
+vcluster get service-cidr
+10.96.0.0/12
+#######################################################
+	`,
+		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			return cmd.Run(cobraCmd)
+		}}
+
+	return cobraCmd
+}
+
+func (cmd *serviceCIDRCmd) Run(cobraCmd *cobra.Command) error {
+	// first load the kube config
+	kubeClientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(clientcmd.NewDefaultClientConfigLoadingRules(), &clientcmd.ConfigOverrides{
+		CurrentContext: cmd.Context,
+	})
+
+	// load the rest config
+	kubeConfig, err := kubeClientConfig.ClientConfig()
+	if err != nil {
+		return fmt.Errorf("there is an error loading your current kube config (%v), please make sure you have access to a kubernetes cluster and the command `kubectl get namespaces` is working", err)
+	}
+
+	client, err := kubernetes.NewForConfig(kubeConfig)
+	if err != nil {
+		return err
+	}
+
+	if cmd.Namespace == "" {
+		cmd.Namespace, _, err = kubeClientConfig.Namespace()
+		if err != nil {
+			return err
+		} else if cmd.Namespace == "" {
+			cmd.Namespace = "default"
+		}
+	}
+
+	// make sure namespace exists
+	_, err = client.CoreV1().Namespaces().Get(context.Background(), cmd.Namespace, metav1.GetOptions{})
```

Comment:
we don't want to enforce this here, as creating a namespace shouldn't be part of this command, so just delete this block